lib/routemap.c

```diff
@@ -854,11 +854,13 @@ static void vty_show_route_map_entry(struct vty *vty, struct route_map *map)
 	struct route_map_index *index;
 	struct route_map_rule *rule;
 
-	vty_out(vty, "%s:\n", frr_protonameinst);
+	vty_out(vty, "route-map: %s Invoked: %" PRIu64 "\n",
+		map->name, map->applied);
 
 	for (index = map->head; index; index = index->next) {
-		vty_out(vty, "route-map %s, %s, sequence %d\n", map->name,
-			route_map_type_str(index->type), index->pref);
+		vty_out(vty, " %s, sequence %d Invoked %" PRIu64 "\n",
```

Comment:
Shouldn't this be preference instead of sequence ?

Reply:
It's called "sequence" in the CLI help text too:
```
DEFUN_NOSH (route_map,
       route_map_cmd,
       "route-map WORD <deny|permit> (1-65535)",
...
       "Sequence to insert to/delete from existing route-map entry\n")
```
I think sequence makes sense too since we're applying route-map entries "in order of the sequence".

+ If we change it, please change it everywhere :)

Reply:
I didn't write the code :) . I'll fix it.